commerce/bookstore/README.md

```diff
@@ -21,7 +21,8 @@ You can find the implementation of each function listed above in the source code
 
 * [TypeDB](https://docs.vaticle.com/docs/running-typedb/install-and-run) v2.14.1+
 * Python v.3.9+
-* `typedb.client` — [Python client](https://docs.vaticle.com/docs/client-api/python) for TypeDB
+  * `typedb.client` — [Python client](https://docs.vaticle.com/docs/client-api/python) for TypeDB
+  * Common Python libraries: csv, argparse, uuid, random
```

Comment:
Very good to include this. This - as well as the addition of a `BUILD` file - reminds me that every other example, currently, is a Bazel project. We most likely do not want to diverge from this.

Converting a Python project to a Bazel project is very easy. See https://github.com/vaticle/typedb-examples/tree/master/telecom/phone_calls/python for inspiration. Basically we just need to add some `py_binary` and `py_library` targets to our `BUILD` file.

- `py_binary` represents an entry point; a Python script that the user is expected to run.
- `py_library` represents a library of Python functions; a Python script that is expected to be only referenced through "import" statements, and not run as a standalone binary.

So for the current project, `config` and `loaders` would be libraries, and `load_data` and `requests` would be binaries.

You can pretty much copy the `py_library` and `py_binary` syntax from this `BUILD` file: https://github.com/vaticle/typedb-examples/blob/master/telecom/phone_calls/python/BUILD

The other thing to do is to specify these requirements (csv, argparse, uuid, random) in a `requirements.txt` file. I believe `bazel run //commerce/bookstore:queries` will then automatically download these dependencies and just run - the "one-click run" that Bazel gives us.

Reply:
I have tried fixing this. But so far was unsuccessful. I will try again, maybe do some quickstart with Bazel. So far our Build files seem easy to understand but are still very tricky to make it work somehow.
So far only the checkstyle test working and I have already updated the files to pass this test. I'm considering adding unit tests to the example as it seems an easier task than making Bazel work so far.

Reply:
I have managed to make this work https://github.com/vaticle/typedb-examples/pull/143/commits/240df54e638d9a14bbde669a8d4190cad182be6f

Reply:
Why would we need to do that? It's just an extra step. So the onboarded engineer would need to learn TypeDB and Bazel. That seems counter-productive to me. Like the very opposite of what we should do in the example.

Adding sophistication to example documentation is very undesirable.

Reply:
This conversation has not aged well. Marking as resolved since no more Bazel usage!